keyboards/projectcain/relic/keymaps/default/config.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2021 projectcain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define ENCODERS_PAD_A { D4 }
+#define ENCODERS_PAD_B { D6 }
```

Comment:
Is there a reason that this isn't defined at the kb level?

Reply:
Not necessarily - I can definitely move it

Reply:
For stuff like this, it's best to have hardware configuration in the keyboard's config.h, rather than at keymap level, unless it's a hardware mod (eg, not in the normal design)